arduino/include/arduino.h

```diff
@@ -0,0 +1,35 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+#include "serialib.h"
+#include <ios>
+class Arduino
```

Comment:
add a new line between the includes and the class declaration

Reply:
!

Reply:
why you didn't do it?

Reply:
fixed